.github/workflows/docs.yml

```diff
@@ -12,7 +16,8 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: '3.9'
```

Comment:
Should the CI run on Python 3.10 instead of 3.9 as it is current stable version and 3.11 releases soon?

Reply:
@vinitkumar RTD doesn't have 3.10 available so I'm trying to replicate the build env on there.